lib/plugins/aws/deployFunction/index.js

```diff
@@ -173,8 +173,11 @@ class AwsDeployFunction {
             throw new this.serverless.classes.Error(errorMessage);
           }
 
-          if (typeof params.Environment.Variables[key] !== 'string') {
-            params.Environment.Variables[key] = _.toString(params.Environment.Variables[key]);
+          if (
+            typeof params.Environment.Variables[key] !== 'string' &&
+            params.Environment.Variables[key]
```

Comment:
Let's change condition to simply `if (params.Environment.Variables[key] != null)`

Note that above will ignore `0` and that will introduce a regression

Reply:
So simple is
```
          if (params.Environment.Variables[key]) {
            params.Environment.Variables[key] = String(params.Environment.Variables[key]);
          }
```
I will change that

Reply:
`if (params.Environment.Variables[key])` won't pick `0` and some other falsy values. 

To work properly it has to be `if (params.Environment.Variables[key] != null)` (that way we only discard _no values_ as `null` and `undefined`)

Reply:
@medikoo I updated. Pls check it!